docs/snippets/collecting-metrics.md

```diff
@@ -47,27 +47,71 @@ aggregating timeseries metrics and alerting. It can also be used to scrape the O
     kubectl apply -f https://raw.githubusercontent.com/knative-extensions/monitoring/main/servicemonitor.yaml
     ```
 
-1. Grafana dashboards can be imported from the [`knative-extensions` repository](https://github.com/knative-extensions/monitoring/tree/main/grafana).
+### Access the Prometheus instance locally
+
+By default, the Prometheus instance is only exposed on a private service named `prometheus-operated`.
+
+To access the console in your web browser:
+
+1. Enter the command:
+
+    ```bash
+    kubectl port-forward -n default svc/prometheus-operated 9090
+    ```
+
+1. Access the console in your browser via `http://localhost:9090`.
+
+### Setting up Grafana
+
+1. Grafana dashboards can be imported from the [`monitoring` repository](https://github.com/knative-extensions/monitoring/tree/main/grafana).
 
 1. If you are using the Grafana Helm Chart with the Dashboard Sidecar enabled, you can load the dashboards by applying the following configmaps.
 
     ```bash
     kubectl apply -f https://raw.githubusercontent.com/knative-extensions/monitoring/main/grafana/dashboards.yaml
     ```
 
-### Access the Prometheus instance locally
+    !!! caution
+        You will need to ensure that the helm chart has following values configured, otherwise the dashboards loading will not work.
+        ```yaml
+        grafana:
+          sidecar:
+            dashboards:
+              enabled: true
+              searchNamespace: ALL
+        ```
+        If you have an existing configmaps check the value of `grafana_dashboard` label. In case this is `true`, add the `labelValue: true` attribute to the helm chart above.
+        ```bash
+        kubectl -n knative-serving get configmaps/knative-serving-dashboards -o jsonpath="{.metadata.labels.grafana_dashboard}"; echo
+        ```
 
-By default, the Prometheus instance is only exposed on a private service named `prometheus-operated`.
+#### Access the Grafana instance locally
 
-To access the console in your web browser:
+By default, the Grafana instance is only exposed on a private service named `prometheus-grafana`.
+
+To access the dashboards in your web browser:
 
 1. Enter the command:
 
     ```bash
-    kubectl port-forward -n default svc/prometheus-operated 9090
+    kubectl port-forward -n default svc/prometheus-grafana 3000:80
     ```
 
-1. Access the console in your browser via `http://localhost:9090`.
+1. Access the dashboards in your browser via `http://localhost:3000`.
+
+1. Use the default credentials to login:
+   
+    ```text
+    username: admin
+    password: prom-operator
+    ```
+
+    !!! tip
```

Comment:
Although this might be useful if password changes etc (I know I mentioned it on the other PR),  let's omit it for now as it seems a bit not that Knative related, we are going a bit too far.

Reply:
I can try to find one that suits us!

Reply:
I did some research, and in my opinion the most useful link is https://github.com/prometheus-community/helm-charts/blob/main/charts/kube-prometheus-stack/README.md (in particular the [Configuration](https://github.com/prometheus-community/helm-charts/blob/main/charts/kube-prometheus-stack/README.md#configuration) section).

There is also a [Further Information](https://github.com/prometheus-community/helm-charts/blob/main/charts/kube-prometheus-stack/README.md#further-information) section where they linked external resources like Prometheus Operator, Prometheus and Grafana.

What do you think?

Reply:
We could add a pointer to that instead of the tip, but it does not say much about credentials and not sure if it is going to help a novice. So for now let's not add anything, remove the tip and in the future we can always revise it.

Reply:
Done with the latest commit 😄